orchestra/communication/utils.py

```diff
@@ -27,7 +27,8 @@ def mark_worker_as_winner(worker, task, required_role_counter,
     else:
```

Comment:
do we have to check that the staffing request is still valid?

on a related note: is there a test that ensures that if you're the first winner of a request, but that request is already closed, you don't win?

Reply:
Yup